.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,8 +13,15 @@ one.
 <!--
 Please let us know what behaviour you expected and how Upbound diverged from
 that behaviour.
+
+If this bug pertains to an existing epic please cross-reference it below, for
+example by writing:
+
+Epic https://github.com/upbound/some-repo/issues/1000
 -->
 
+Epic #
```

Comment:
Did we decide to link to closed source repos (for the epics) from open source repos?

Reply:
My thinking here is that our options are:

1. Prompt to link to an epic, which in some cases might result in an unfollowable link _if_ the issue is raised in a public repo but the epic is in a private repo.
1. Add a check box with something like "go find the relevant epic and update it to point here".
1. Do nothing.

IMO the prompt has the highest likelihood of being a process we actually follow. I'm pretty skeptical that we're really going to go maintain the habit of finding the epic and updating a table of issues. On the other hand, this process is no more onerous than using the "Fixes #1234" syntax we regularly use to close PRs, and it at least creates cross-links/breadcrumbs from the epic to the story even if they are down mixed into the commentary and not in a neatly discoverable table.

So in short I personally suggest this tradeoff.

Reply:
@negz that sounds reasonable to me, if it becomes a problem in the future for some reason we can always invest in additional tooling :+1: